crates/proposer-service/src/tests/mod.rs

```diff
@@ -80,76 +84,6 @@ async fn test_proposer_service() {
     assert_eq!(response.start_block, 0);
 }
 
-#[tokio::test]
-async fn test_vkey_hash_mismatch() {
```

Comment:
any drawback to keep this test?

Reply:
It tests for an error that can no longer occur. Possibly adapting it to check an invalid proof is correctly detected during the verify step might make sense.

Reply:
@iljakuklic I'll leave the test stub, we already have the thicket for tests https://github.com/agglayer/provers/issues/96